koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -270,12 +270,14 @@ def get_expected_nvr(self):
     def format_label(self, label_id):
         """Formats string with user-facing LABEL name and its alternatives"""
 
-        assert label_id in LABEL_NAME_MAP
+        if not label_id in LABEL_NAME_MAP:
+            dockerfile_parse.parser.logger.error('"%s" does not contain "%s"',
+                                                 LABEL_NAME_MAP, label_id)
         label_map = LABEL_NAME_MAP[label_id]
         if len(label_map) == 1:
             return label_map[0]
         else:
-            return "%s (or %s)" % (label_map[0], " or ".join(label_map[1:]))
+            return "%s (or %s)" % (label_map[0], " ".join(label_map[1:]))
```

Comment:
Why?

Reply:
Unless I'm reading the original wrong, it will print 'or' multiple times (needlessly, or maybe not?)

Reply:
current: `A (or B or C or D)`
this PR: `A (or B C D)`

How is this related to `Remove runtime assertions`?

Reply:
I see. Will revert.